### PR TITLE
ENG-2566 Refactored users-roles-ui.spec.js

### PR DESCRIPTION
**Pull request by g-di-martino**
Added editorconfig to enforce newline at file end on save
Updated eslint pre-commit hook to lint only staged files
Refactored tests to use page object pattern
Refactored tests to perform set-up and tear-down using REST APIs

### CODE REVIEW

File: ui-tests/cypress/support/command/validation-commands.js [hunk 1]
```diff
@@ -14,4 +14,15 @@ Cypress.Commands.add('validateToast', ( page, isOk = true , text) => {
...
+ * @param list - the list of html elements to be validated
+ * @param values - the ordered expected texts
+ */
+Cypress.Commands.add('validateListTexts', (list, values) => {
+  for (let i = 0; i < values.length; i++) {
+    cy.get(list).eq(i).should('have.text', values[i]);
+  }
+});
+
+export {};
```
Comment (ichalagashvili) [inline ui-tests/cypress/support/command/validation-commands.js:28]: new line please

File: ui-tests/cypress/support/pageObjects/app/Content.js [hunk 2]
```diff
@@ -24,9 +24,7 @@ export default class Content extends WebElement {
   getTitle() {
     return this.getContents()
                .children(htmlElements.div).eq(1)
-               .children(htmlElements.div)
-               .children(htmlElements.div)
-               .children(htmlElements.h1);
+               .find(htmlElements.h1);
   }
 
 }
```
Comment (ichalagashvili) [inline ui-tests/cypress/support/pageObjects/app/Content.js:30]: new line please

File: ui-tests/package.json [hunk 3]
```diff
@@ -50,4 +51,4 @@
       }
     }
   }
-}
+}
```
Comment (ichalagashvili) [inline ui-tests/package.json:54]: new line please

**Review by ichalagashvili — APPROVED** [review]

**Review by jeffgo10 — APPROVED** [review]
lintStaged looks good. would try that approached on other projects as well. good one G

**Pull request merged**